packages/create-message-kit/index.js

```diff
@@ -14,12 +14,21 @@ const packageJson = JSON.parse(
   fs.readFileSync(resolve(__dirname, "package.json"), "utf8"),
 );
 const version = packageJson.version;
-
 program
   .name("byob")
   .description("CLI to initialize projects")
   .action(async () => {
-    intro(pc.red(`Welcome to MessageKit CLI v${version}!`));
+    log.info(pc.cyan(`Welcome to MessageKit CLI v${version}!`));
+    const coolLogo = `
+███╗   ███╗███████╗███████╗███████╗ █████╗  ██████╗ ███████╗██╗  ██╗██╗████████╗
+████╗ ████║██╔════╝██╔════╝██╔════╝██╔══██╗██╔════╝ ██╔════╝██║ ██╔╝██║╚══██╔══╝
+██╔████╔██║█████╗  ███████╗███████╗███████║██║  ███╗█████╗  █████╔╝ ██║   ██║   
+██║╚██╔╝██║██╔══╝  ╚════██║╚════██║██╔══██║██║   ██║██╔══╝  ██╔═██╗ ██║   ██║   
+██║ ╚═╝ ██║███████╗███████║███████║██║  ██║╚██████╔╝███████╗██║  ██╗██║   ██║   
+╚═╝     ╚═╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝ ╚═════╝ ╚══════╝╚═╝  ╚═╝╚═╝   ╚═╝   
+Powered by XMTP`;
```

Comment:
nit: this is indeed cool, but it's a bit hard to read. is there another text style that would work better?